### DISCUSSION TITLE
data import

### DISCUSSION THREAD

**Discussion by taunusrenner** [mytourbook/mytourbook#921] [General]

**Comment by FJBDev** [comment]
Can you give more info such as screenshots, detailed scenario and maybe also attach the problematic files in order to reproduce the issue locally ?

**Comment by wolfgang-ch** [comment]
@taunusrenner 

Your original content is removed, should this discussion be deleted ?

![image](https://user-images.githubusercontent.com/1283445/188364745-e7900d3a-6418-41a1-8d43-ceacd5828519.png)

![image](https://user-images.githubusercontent.com/1283445/188364600-47c932b9-fa17-4a4e-8680-70ad8d1fe2b0.png)
